horovod/common/operations.cc

```diff
@@ -1092,6 +1093,21 @@ void BackgroundThreadLoop(HorovodGlobalState& state, MPIContext& ctx) {
            "allgather and hierarchical allreduce.";
   }
 
+#if HAVE_CUDA
+  // Set number of CUDA streams to use
+  auto horovod_num_streams =
+      std::getenv("HOROVOD_NUM_STREAMS");
```

Comment:
Could you make a constant in operations.h, and rename to `HOROVOD_NUM_NCCL_STREAMS` (rename variable as well)?

Reply:
Done. I also changed the names of the new variables in the global state struct from `num_streams` -> `num_nccl_streams` and `current_stream` to `current_nccl_stream`.